### PR TITLE
Track copied text template

### PR DESCRIPTION
**Pull request by mgonera**
Hi, first simple tag for review :) it's purpose is to track custom events with texts that are copied by people browsing the website.

### CODE REVIEW

File: src/trackCopiedText.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/**
+ * Tracking text copied by the user.
+ * 
+ * Adds event listener
+ */
+
+ export default (subscribe) => {
+    window.addEventListener("copy",function(e){
```
Comment (ArekZc) [inline src/trackCopiedText.js:8]: use array function in this file

Comment (ArekZc) [reply src/trackCopiedText.js:8]: when you use addEventListener, callback function should be named function like:

```
window.addEventListener('copy', processCopyEvent)
```

File: src/trackCopiedText.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        ppas_processCopyEvent(e);
+    });
+    
+    const ppas_processCopyEvent = function(e){
```
Comment (ArekZc) [inline src/trackCopiedText.js:12]: do not use prefixes like ppas, whole file has already isolated scope

Comment (ArekZc) [reply src/trackCopiedText.js:12]: try to always use camelCase notation to be consistent with the rest of the code

File: src/trackCopiedText.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        let copiedItemText = getSelectedText();
```
Comment (ArekZc) [inline src/trackCopiedText.js:13]: you shouldn't use a function before it's defined

File: src/trackCopiedText.js (hunk 1 above)
Comment (mgonera) [reply src/trackCopiedText.js:8]: Done

File: src/trackCopiedText.js (hunk 2 above)
Comment (mgonera) [reply src/trackCopiedText.js:12]: Sure, this came from the fact that I literally copied the solution that already existed ;)

File: src/trackCopiedText.js (hunk 3 above)
Comment (mgonera) [reply src/trackCopiedText.js:13]: Done

File: templates.js [hunk 4]
```diff
@@ -162,13 +162,24 @@ ${fs.readFileSync(path.join(__dirname, 'build/collectHeatmapClicks.js'), { encod
...
+    id: 'trackCopiedText',
+    name: 'Track copied text',
+    description: `
+    This template allows you to track pieces of text that are copied to clipboard by your website users.
+    `,
+    template: `
+${fs.readFileSync(path.join(__dirname, 'build/trackCopiedText.js'), { encoding: 'utf-8' })}
+
+trackCopiedText(function (copiedItemText) {
+_paq.push(["trackEvent","User interaction","Copying text",copiedItemText]);
```
Comment (rezik007) [inline templates.js:179]:
```suggestion
  window._paq.push(['trackEvent', 'User interaction', 'Copying text', copiedItemText]);
```
To keep convention from other code examples.

**Review by rezik007 — APPROVED** [review]

**Pull request merged**